src/drt/src/gc/FlexGC_init.cpp

```diff
@@ -355,6 +412,46 @@ void FlexGCWorker::Impl::initDRWorker()
     addNonTaperedPatches(gNet, uDRNet.get());
   }
 }
+void FlexGCWorker::Impl::initNetsFromDesign(const frDesign* design)
+{
+  int cnt = 0;
+  auto block = design->getTopBlock();
+  for (auto& net : block->getNets()) {
+    // always first generate gcnet in case owner does not have any object
+    auto it = owner2nets_.find(net.get());
+    if (it == owner2nets_.end()) {
+      addNet(net.get());
```

Comment:
Why add every net even if it has no shapes in the checking region?  Should this be delayed to initRouteObj?

Reply:
I don't think it has high cost anyway. In main, if the net doesn't have any shapes it will be skipped.